modules/core/src/lib/deck.ts

```diff
@@ -358,7 +359,8 @@ export default class Deck {
    */
   private _lastPointerDownInfo: PickingInfo | null = null;
 
-  constructor(props: DeckProps) {
+  constructor(props: DeckProps<ViewsT>) {
+    // @ts-ignore views
```

Comment:
What are we ignoring about views type here?

Reply:
The default `views` value is `null` but may not match user-provided `ViewsT`